### DISCUSSION TITLE
Sortin labels

### DISCUSSION THREAD

**Discussion by manuelpmgt** [rappasoft/laravel-livewire-tables#1650] [General]
Hi, I am working on a project and i have columns with names and numbers. Is there a way to change the sorting labels on a column basis? I mean if I sort a name column it should show "Applied Sorting: Field A-Z" but if i sort a number column it should show "Applied Sorting: Field ASC.

**Comment by lrljoe** [comment]
Yes, on each column you have this method to override what is displayed when sorting:

```
->setSortingPillDirections('Asc', 'Desc')
```

**Reply by manuelpmgt** [reply]
Thank you! I thought that you could only set it globally

**Reply by lrljoe** [reply]
No problem, if you can mark it as answered then that'll help others looking for the same thing! :)
